src/common/json_output.rs

```diff
@@ -185,17 +204,32 @@ pub struct DescribeEifInfo {
     #[serde(rename = "SignatureCheck")]
     /// Specifies if the EIF signature check passed.
     pub sign_check: Option<bool>,
+    #[serde(rename = "ImageName")]
+    /// Assigned or default EIF name
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub img_name: Option<Value>,
+    #[serde(rename = "ImageVersion")]
+    /// Assigned or default EIF version
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub img_version: Option<Value>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(rename = "Metadata")]
+    /// EIF metadata
+    pub metadata: Option<DescribeMetadata>,
 }
 
 impl DescribeEifInfo {
     /// Create describe information structure for EIF.
     pub fn new(
-        version: u16,
+        version: String,
```

Comment:
I'm not sure why this change is here.

Reply:
True, looks like no big reason to change this type

Reply:
reverted